.pipeline/lib/config.js

```diff
@@ -0,0 +1,52 @@
+const options = require('@bcgov/pipeline-cli').Util.parseArguments();
+
+const changeId = options.pr;
+const name = 'bcat';
+const version = '1.0.0';
+let shortSha;
+
+if (changeId && typeof changeId === 'string') {
+  shortSha = changeId.slice(0, 7);
+}
+
+const updatedChangeId = shortSha || changeId;
+
+Object.assign(options.git, { owner: 'bcgov', repository: 'BCAT' });
+const phases = {
+  build: {
+    changeId: updatedChangeId,
+    instance: `${name}-build-${updatedChangeId}`,
+    name: `${name}`,
+    namespace: 'bfe2da-tools',
```

Comment:
Could we assign `bfe2da` to a variable

Reply:
This might be easier to keep track of. I can update that, thanks for suggestion!